AirCasting/AirCastingApp.swift

```diff
@@ -8,22 +8,25 @@
 import SwiftUI
 import Firebase
 import Combine
+
 @main
 struct AirCastingApp: App {
     @Environment(\.scenePhase) var scenePhase
     private let authorization: UserAuthenticationSession
     private let syncScheduler: SynchronizationScheduler
     private let microphoneManager: MicrophoneManager
+    private let appState: AppState
     private var sessionSynchronizer: SessionSynchronizer
     private let persistenceController = PersistenceController.shared
     private let appBecameActive = PassthroughSubject<Void, Never>()
     @ObservedObject private var offlineMessageViewModel: OfflineMessageViewModel
     private var cancellables: [AnyCancellable] = []
-
+//    let appState = AppState()
+    
     init() {
-        #if !DEBUG
+//        #if !DEBUG
         FirebaseApp.configure()
-        #endif
+//        #endif
```

Comment:
Why did you comment `#if !DEBUG` and `#endif`?

Reply:
same: I cannot see it, maybe from earlier commit? 🙁

Reply:
it's weird, why github is marking it as changes if it's different now. Sorry, I can see in the code that it's not commented anymore 🤷🏻‍♀️

Reply:
Important that we agreed on this 😛